core/src/main/scala/tech/beshu/ror/accesscontrol/factory/decoders/definitions/LdapServicesDecoder.scala

```diff
@@ -217,19 +213,37 @@ object LdapServicesDecoder {
       }
       .mapError(DefinitionsLevelCreationError.apply)
 
-  private def hostConnectionErrorDecodingFailureFrom(error: HostConnectionError) = {
-    val connectionErrorMessage = Message(
-      s"There was a problem with LDAP connection to: ${error.hosts.map(_.url.toString()).toList.mkString(",")}"
-    )
-    DecodingFailureOps.fromError(DefinitionsLevelCreationError(connectionErrorMessage))
-  }
-
-  private def serverDiscoveryConnectionDecodingFailureFrom(error: ServerDiscoveryConnectionError) = {
-    val connectionErrorMessage = Message(
-      s"There was a problem with LDAP connection in discovery mode. " +
-        s"Connection details: recordName=${error.recordName.getOrElse("default")}, " +
-        s"providerUrl=${error.providerUrl.getOrElse("default")}")
-    DecodingFailureOps.fromError(DefinitionsLevelCreationError(connectionErrorMessage))
+  private def connectionErrorDecodingFailureFrom(error: ConnectionError) = {
+    def connectionErrorFrom(message: Message) = {
+      DecodingFailureOps.fromError(DefinitionsLevelCreationError(message))
```

Comment:
WDYT about passing the message as String and creating the Message instance here?